cmd/agent/receiver.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/kcarretto/paragon/agent"
+	"github.com/kcarretto/paragon/api/codec"
+	"github.com/kcarretto/paragon/script"
+	"github.com/kcarretto/paragon/script/stdlib"
+	"go.uber.org/zap"
+)
+
+type Receiver struct {
+	context.Context
+
+	Log *zap.Logger
+}
+
+func (r Receiver) Receive(w agent.MessageWriter, msg agent.ServerMessage) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method Receiver.Receive should have comment or be unexported